### DISCUSSION TITLE
Alternate mod should be compatible with osu!taiko

### DISCUSSION THREAD

**Discussion by ianpep** [ppy/osu#18086] [Ideas]
Alternate mod is currently not available in osu!taiko, although it feels like it should be. When using the mod, players would be forced to swap hands each hit, similarly to how it functions in standard at the moment.

**Comment by bdach** [comment]
That was attempted and then was promptly swamped because people wanted it to work properly with kddk and kdkd playstyles, which then complicated the implementation which ended up being unreadable.

Paper trail starts here: https://github.com/ppy/osu/pull/7953#issuecomment-591357112
